insights/combiners/tests/test_krb5.py

```diff
@@ -0,0 +1,90 @@
+from insights.combiners.krb5 import AllKrb5Conf
+from insights.parsers.krb5 import Krb5Configuration
+from insights.tests import context_wrap
+
+
+KRB5CONFIG = """
+# Configuration snippets may be placed in this directory as well
+includedir /etc/krb5.conf.d/
+include /etc/krb5test.conf
+module /etc/krb5test.conf:residual
+
+[logging]
+ default = FILE:/var/log/krb5libs.log
+ kdc = FILE:/var/log/krb5kdc.log
+
+[realms]
+ dns_lookup_realm = false
+ default_ccache_name = KEYRING:persistent:%{uid}
+ default_ccache_name2 = KEYRING:%{uid}:persistent
+ kdc_default_options = default.example.com
+ kdc_default_options = default2.example.com
+ EXAMPLE.COM = {
+  kdc = kerberos.example.com
+  admin_server = kerberos.example.com
+  auth_to_local = RULE:[1:$1@$0](.*@.*EXAMPLE.ORG)s/@.*//
+ }
+ EXAMPLE4.COM = {
+  kdc = kerberos.example4.com
+  admin_server = kerberos.example4.com
+ }
+ ticket_lifetime = 24h
+# renew_lifetime = 7d
+# forwardable = true
+# rdns = false
+""".strip()
+
+
+KRB5CONFIG2 = """
+# Configuration snippets may be placed in this directory as well
+
+[realms]
+ dns_lookup_realm = false
+ ticket_lifetime = 24h
+# default_ccache_name = KEYRING:persistent:%{uid}
+ EXAMPLE.COM = {
+  kdc = kerberos.example.com
+  kdc = test2.example.com
+  kdc = test3.example.com
+  admin_server = kerberos.example.com
+ }
+
+[logging]
+ default = FILE:/var/log/krb5libs2.log
+ kdc = FILE:/var/log/krb5kdc2.log
+ admin_server = FILE:/var/log/kadmind2.log
+
+[libdefaults]
+ dnsdsd = false
+ tilnvs = 24h
+ default_ccache_name = KEYRING:%{uid}:persistent
+ EXAMPLE2.COM = {
+  kdc = kerberos.example2.com
+  admin_server = kerberos.example2.com
+ }
+ EXAMPLE3.COM = {
+  kdc = kerberos.example3.com
+  admin_server = kerberos.example3.com *
+ }
+""".strip()
+
+
+def test_active_krb5_nest():
```

Comment:
Does this test how the included file overrides the main `/etc/krb5.conf` file?  I can't see how it does...

Reply:
The `assert not result.has_option("logging", "admin_server")` verified it.